ogr/abstract.py

```diff
@@ -1688,6 +1688,18 @@ def get_pr(self, pr_id: int) -> "PullRequest":
         """
         raise NotImplementedError()
 
+    def get_pr_diff(self, pr_id: int) -> dict:
+        """
+        Get files diff of a pull request.
+
+        Args:
+            pr_id: ID of the pull request.
+
+        Returns:
+            Dictionary representing files diff.
+        """
+        raise NotImplementedError()
```

Comment:
```suggestion
    def get_pr_files_diff(self, pr_id: int) -> dict:
        """
        Get files diff of a pull request.

        Args:
            pr_id: ID of the pull request.

        Returns:
            Dictionary representing files diff.
        """
        raise NotImplementedError()
```

I would add a `file` into the name to not confuse it with the `diff` (~ patch) of the PR itself.